tests/testthat/apps/app/app.R

```diff
@@ -0,0 +1,3 @@
+library(informalr)
+
+app()
```

Comment:
Hmmm, I would naively think that calling `informalr::app()` is as convenient as running this script with `Rscript  tests/testthat/apps/app/app.R`. I guess this is for Shinytest so I guess it's perfect :grin: :+1:

Reply:
I am also quite confused about this... the app.R calls the app() function, then the shinytest.R calls the app by calling app.R... kind of redundant, but let's leave it since it was something automatically created by the shinytest package :)